trajectory_executor/src/main.cpp

```diff
@@ -14,20 +14,20 @@
  * the License.
  */
 
-#include <ros/ros.h>
-#include "trajectory_executor/trajectory_executor.hpp"
+#include <rclcpp/rclcpp.hpp>
+#include "trajectory_executor/trajectory_executor_node.hpp"
 
-/**
- * Main application entry point for TrajectoryExecutor Node
- */
-int main(int argc, char** argv) 
+int main(int argc, char **argv) 
 {
-  ros::init(argc, argv, "trajectory_executor");
-  ROS_DEBUG("Entered main function for Trajectory executor, starting node...");
-  trajectory_executor::TrajectoryExecutor te;
+  rclcpp::init(argc, argv);
+
+  auto node = std::make_shared<trajectory_executor::TrajectoryExecutor>(rclcpp::NodeOptions());
+  
+  rclcpp::executors::MultiThreadedExecutor executor;
```

Comment:
It looks like we do indeed have multithreading here, is there a reason this changed from the original trajectory executor implementation?

Reply:
This is part of the initial ROS2 code contained our template_package main.cpp file: https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/template_package/src/main.cpp

Do you think this needs to be updated to match the original trajectory executor implementation?